content/influxdb/v2.0/reference/beta-17-upgrade-guide.md

```diff
@@ -0,0 +1,275 @@
++-----+------------------------+----+---------------+---------+-------------+
+​
+BUCKETS    +add | -remove | unchanged
++-----+------------------------+----+---------------+------------------+-------------+
+| +/- |     METADATA NAME      | ID | RESOURCE NAME | RETENTION PERIOD | DESCRIPTION |
++-----+------------------------+----+---------------+------------------+-------------+
+| +   | fasting-taussig-c9c007 |    | apps          | 0s               |             |
++-----+------------------------+----+---------------+------------------+-------------+
++-----+------------------------+----+---------------+------------------+-------------+
+| +   | great-davinci-c9c005   |    | new_telegraf  | 0s               |             |
++-----+------------------------+----+---------------+------------------+-------------+
++-----+------------------------+----+---------------+------------------+-------------+
+| +   | stubborn-hugle-c9c003  |    | telegraf      | 719h59m59s       |             |
++-----+------------------------+----+---------------+------------------+-------------+
+|                                                          TOTAL       |      3      |
++-----+------------------------+----+---------------+------------------+-------------+
+```
+
+Now you have all the resources from your old instance stored in your new instance.
+Log into your new instance (probably http://localhost:8086) and take a look.
+
+## 8. Setup the integrations to point to new instance
+
+Now is a good time to set up any integrations you needed to disable before this process all started.
+You have a better understanding of what those might be, but anything related to Telegraf, client libraries, custom applications, or 3rd party sinks will need to be setup again using new tokens and credentials.
+
+{{% note %}}
+*So far, we haven’t moved any time series data over.*
+If you need to load your existing historical data into your instance, keep reading.
+{{% /note %}}
+
+## 9. Load your historical data into new instance
+
+There’s no direct migration of raw storage files, but it’s straightforward to use the command line to export and then re-import your data.
+You can dump the raw Flux result using the `influx query` command.
+For the time range, you can just pick something before your bucket’s retention period, or something a really long time ago if you have an unlimited retention period.
+
+```sh
+influx query -c influx_old ‘from(bucket: “my-bucket”) |> range(start: -3y)’ >  my-bucket.csv
+```
+
+```sh
+influx write -c default --format csv -b my-bucket -f my-bucket.csv
+```
+
+Repeat that process for each bucket.
+
+## Wrapping up
+
+You should now have the latest version of InfluxDB running with all your resources, data, and integrations configured.
+Double check and make sure everything is there and it is working as expected.
+Once you have everything set up again, you can safely turn off your old instance and archive that data directory.
```

Comment:
To verify the latest....

Wrapping up >> Verify InfluxDB resources, data, and integration